frontend/src/CreateAccount

```diff
@@ -0,0 +1,59 @@
+import React, {useState} from 'react';
+
+function AccountForm(props) {
+  const [person, setPerson] = useState(
+     {
+        username: "",
+        email: "",
+        password: "",
+     }
+  );
+
+  function handleChange(event) {
+    const { username, value } = event.target;
+    if (username === "password")

```

Comment:
Why does the conditional check to see if username === "password"

Reply:
@RandallCaler

Reply:
it looks like 'username' is just a generic variable that could be passed either a username or a password. maybe a different name for it would be more clear